samcli/commands/package/core/options.py

```diff
@@ -0,0 +1,68 @@
+"""
+Package Command Options related Datastructures for formatting.
+"""
+from typing import Dict, List
+
+from samcli.cli.core.options import ALL_COMMON_OPTIONS, add_common_options_info
+from samcli.cli.row_modifiers import RowDefinition
+
+# The ordering of the option lists matter, they are the order in which options will be displayed.
+
+REQUIRED_OPTIONS: List[str] = ["s3_bucket", "resolve_s3"]
+
+AWS_CREDENTIAL_OPTION_NAMES: List[str] = ["region", "profile"]
+
+INFRASTRUCTURE_OPTION_NAMES: List[str] = [
+    "s3_prefix",
```

Comment:
How can we make sure these are not duplicated between the command itself and its help text? We could at least think about adding generic unit tests which will do a cross check if certain parameter is defined in one place and missed in other.

Reply:
https://github.com/aws/aws-sam-cli/pull/5298/files#diff-484e02a4e1f54c1be185a1458fbb5d53b4bc9f4be8b4fd73006f27add3dcdd21R10